mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +26,41 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should image open?
+   */
+  isShouldOpen?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
+
+  const [isOpen, setIsOpen] = useState(false);
+
   const className = clsx(styles.image, props.className);
 
+  /**
+   * Function full screen image
+   */
+  const onClickHandler = (value: boolean) => {
+    setIsOpen(value);
+  };
```

Comment:
Maybe better don't use additional wrapper and use setIsOpen inside code instead of function-wrapper? What do you think?

Reply:
Ok. I will change

Reply:
@Ekaterina1994 changed